WeatherForecast/WeatherForecast/Resources/Extensions.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Extensions.swift
+//  WeatherForecast
+//
+//  Created by 윤형석 on 1/30/24.
+//
+
+import Foundation
+import UIKit
+
+extension Date {
+    func toString(format: String) -> String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.locale = .init(identifier: "ko_KR")
+        dateFormatter.dateFormat = format
+        return dateFormatter.string(from: self)
+    }
+    
+    func toString(timeStyle: DateFormatter.Style) -> String {
```

Comment:
```suggestion
    func toString(with style: DateFormatter.Style) -> String {
```
```suggestion
    func toString(withDateStyle style: DateFormatter.Style) -> String {
```

p4;
네이밍에 고민이 있다면 Swift API Design Guidelines 를 따르도록 작성해보는 것을 적극 권장드리고 싶습니다!
[Strive for Fluent Usage](https://www.swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage) 가이드에 따르면 메서드의 이름과 파라미터가 영문법적으로 자연스럽게 읽히는 네이밍을 권장하고 있어 네이밍을 제안과 같이 변경해볼 수 있을 것으로 보입니다 👀